xbmc/powermanagement/windows/Win32PowerSyscall.cpp

```diff
@@ -33,41 +103,142 @@ CWin32PowerSyscall::CWin32PowerSyscall()
 
 bool CWin32PowerSyscall::Powerdown()
 {
-  return CWIN32Util::PowerManagement(POWERSTATE_SHUTDOWN);
+  if (!AdjustPrivileges())
+    return false;
+
+  CLog::LogF(LOGINFO, "Requesting OS shutdown");
+
+  if (g_sysinfo.IsWindowsVersionAtLeast(CSysInfo::WindowsVersionWin8))
+    return InitiateShutdownW(NULL, NULL, 0, SHUTDOWN_HYBRID | SHUTDOWN_INSTALL_UPDATES | SHUTDOWN_POWEROFF,
```

Comment:
Should compare to FALSE and not rely on implicit conversion from BOOL to bool. This goes for the other lines as well